src/cli/embedded_help.rb

```diff
@@ -246,7 +254,7 @@ def embedded_files_relative(path, file_name_regexp)
 
     # Loop through all the files in the embedded system
     matches = []
-    EmbeddedScripting.allFileNamesAsString.split(';').each do |file|
+    EmbeddedScripting.fileNames.each do |file|
```

Comment:
```suggestion
    EmbeddedScripting::fileNames.each do |file|
```

Standardize around `::fileNames`. I know it's functionally equivalent but I find "::" to be clearer in meaning.